### PR TITLE
v_3_2_5: switch to hyperkube v1.10.0

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: v_3_2_5/master_template.go [hunk 1]
```diff
@@ -2298,7 +2298,7 @@ coreos:
       RestartSec=0
       TimeoutStopSec=10
       EnvironmentFile=/etc/network-environment
-      Environment="IMAGE=quay.io/giantswarm/hyperkube:v1.9.5"
+      Environment="IMAGE=quay.io/giantswarm/hyperkube:v1.10.0"
```
Comment (xh3b4sd) [inline v_3_2_5/master_template.go]: We could template this version actually and hard code it in the versioned directories here.

Comment (calvix) [reply v_3_2_5/master_template.go]: :+1:  +1

**Comment by corest** [comment]
lgtm if tested in aws/azure/kvm

**Comment by kopiczko** [comment]
Tested on Azure.

**Comment by kopiczko** [comment]
Closing in favour of https://github.com/giantswarm/k8scloudconfig/pull/346

**Pull request closed without merging**